CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ if (EMSCRIPTEN)
 endif()
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake")
-set(CMAKE_OSX_DEPLOYMENT_TARGET 10.13)
+set(CMAKE_OSX_DEPLOYMENT_TARGET 10.15)
```

Comment:
What sort of issues will this cause? Will the website need updating?

Reply:
I can't find any reference to 10.13 on openttd.org, except for OS version of an old bug report and some IRC log discussion about 10.13 being deprecated by Azure.

The only issue this might cause is that people need to upgrade their macOS version, and that we do not support Apple hardware from before the 2012 models anymore when running macOS. Because Apple does not support older hardware with 10.15. And 10.15 support has already been discontinued by Apple.